apps/coordinator/src/components/ScriptExplorer/IndirectSignatureImporter.jsx

```diff
@@ -30,20 +30,13 @@ class IndirectSignatureImporter extends React.Component {
   }
 
   interaction = () => {
-    const { signatureImporter, network, inputs, outputs } = this.props;
+    const { signatureImporter, network, psbt } = this.props;
     const keystore = signatureImporter.method;
-    const bip32Paths = inputs.map((input) => {
-      if (typeof input.bip32Path === "undefined")
-        return signatureImporter.bip32Path; // pubkey path
-      return `${signatureImporter.bip32Path}${input.bip32Path.slice(1)}`; // xpub/pubkey slice away the m, keep /
-    });
 
     return SignMultisigTransaction({
       keystore,
       network,
-      inputs,
-      outputs,
-      bip32Paths,
+      psbt,
```

Comment:
All interactions can now support building the instance based on a PSBT (trezor needed some surgery and a companion bump to caravan-wallets and caravan-psbt)